packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -364,7 +366,7 @@ export default function PostOptionsMenu({
       });
     }
 
-    if (post?.bookmark) {
+    if (post?.bookmark && showPlusSubscription) {
```

Comment:
If we don't have the Plus experiment I think we should not see the "Move to" item, right?

Reply:
Do we have a modal that says subscribe to plus or something? If so, we can use it when this is clicked.

For now, if there is none, we can move forward, but something nice to let the product team know.

Reply:
Updated as stated on Slack ✔️